lua/compe_omni/init.lua

```diff
@@ -0,0 +1,27 @@
+local compe = require("compe")
+local Source = {}
+
+function Source.new()
+  return setmetatable({}, { __index = Source })
+end
+
+function Source.get_metadata(_)
+  return {
+    priority = 100;
+    dup = 0;
+    menu = '[Omni]';
+  }
+end
+
+function Source.determine(_, context)
+  return compe.helper.determine(context)
+end
+
+function Source.complete(_, context)
+  local fn = vim.bo.omnifunc
+  vim.api.nvim_call_function(fn, { 1, '' })
```

Comment:
The omnifunc design is...

1. First, call omnifunc with `findstart==1` that to determine `pum position`. this is the same with `Source.determine` so should use it on `Source.determine`.
2. Invoke actual completion (it should use on `Source.complete`)